homeassistant/components/knx/factory.py

```diff
@@ -202,6 +203,8 @@ def _create_switch(knx_module: XKNX, config: ConfigType) -> XknxSwitch:
         name=config[CONF_NAME],
         group_address=config[CONF_ADDRESS],
         group_address_state=config.get(SwitchSchema.CONF_STATE_ADDRESS),
+        invert=config.get(SwitchSchema.CONF_INVERT),
+        reset_after=config.get(SwitchSchema.CONF_RESET_AFTER),
```

Comment:
Switches can now
- invert their payload 
- and reseted after a given time

Reply:
Things that can reset after a given time should really be represented as events.

Reply:
If this is for momentary buttons, it should be an event.

Reply:
Yes, that makes sense. I'll remove this one again to not hinder this PR any further (and update the docs).

Reply:
Removed `reset_after`.